src/api/app/controllers/webui/staging/concerns/groups_and_users_cache.rb

```diff
@@ -0,0 +1,32 @@
+module Webui
+  module Staging
+    module Concerns
+      module GroupsAndUsersCache
+        extend ActiveSupport::Concern
+
+        private
+
+        def cache_groups_and_users
```

Comment:
I think this method should be splitted into groups and users to only do one thing. Also it does not really do what it says because users is not really cached.

Reply:
splitting a method in two to always call both of them....